next.config.js

```
@@ -2,12 +2,22 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')();
 const withExportImages = require('next-export-optimize-images');
 const LocalizationGenerator = require('./scripts/localizationGenerator');
 
+require('dotenv/config');
+const dotenvExpand = require('dotenv-expand');
+const env = require('dotenv').config();
+dotenvExpand.expand(env);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)